### DISCUSSION TITLE
how to make text run like Marquee in html

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#1276] [Q&A]
### Question

How to make text run like Marquee direction . And it's similar on tiktok . And happened to the web html

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by lawchin** [comment]
```
import flet as ft

def main(page: ft.Page):

    c = ft.Container(
        width=150,
        height=150,
        bgcolor="blue",
        border_radius=10,
        offset=ft.transform.Offset(-2, 0),
        animate_offset=ft.animation.Animation(1000),
    )

    def animate(e):
        c.offset = ft.transform.Offset(0, 0)
        c.update()

    page.add(
        c,
        ft.ElevatedButton("Reveal!", on_click=animate),
    )

ft.app(target=main)
```

more info about animation here:
https://flet.dev/docs/guides/python/animations
